### DISCUSSION TITLE
No local external storage

### DISCUSSION THREAD

**Discussion by moritzifikation** [nextcloud/all-in-one#972] [Questions] [answered]
Hello, i feel very dumb and need help.
I don't understand how to mount my hdd to aio-nc.
I read the readme like 20x but don't understand how to use the commands -e NEXTCLOUD_MOUNT="/mnt/" and -e NEXTCLOUD_MOUNT="/media/".
How do I edit the mastercontainer? :smiling_face_with_tear:

**Comment by moritzifikation** [comment]
great

**Comment by szaimen** [comment, marked as answer]
See https://github.com/nextcloud/all-in-one#how-to-allow-the-nextcloud-container-to-access-directories-on-the-host
